src/BenchmarkDotNet.Core/Columns/QuantityType.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BenchmarkDotNet.Columns
+{
+    public enum UnitType
+    {
+        None,
```

Comment:
Currently we're using this to help render value in `GetValue`. I'll rename `None` to `Dimensionless` but I wouldn't add other elements until we have a use case for them